src/variables.ts

```diff
@@ -9,11 +9,11 @@
  * 5. image         -> assign it a value of a url of your image or ant image that represents you online
  */
 
-let fullName: string;
-let yearOfBirth: number;
-let hobby: string;
-let funFact: string;
-let image: string;
+let fullName: string = "Lolwa Alromi and Razan Alkhateeb";
+let yearOfBirth: number= 2001;
+let hobby: string= "cooking";
+let funFact: string = "Graduated from KUNIV";
+let image: string ="https://th.bing.com/th?id=OIP.GqGVPkLpUlSo5SmeDogUdwHaHa&w=250&h=250&c=8&rs=1&qlt=90&o=6&pid=3.1&rm=2";
```

Comment:
Make sure you have prettier activated on VSCode. 
How did I know? Because there's a space missing between `=` and `"` 😀